solidity/contracts/stubs/RewardsStub.sol

```diff
@@ -28,10 +28,28 @@ contract RewardsStub is Rewards {
         receiveReward(bytes32(i));
     }
 
+    function receiveRewards(uint256[] memory identifieres) public {
```

Comment:
We should consider adding those auxiliary functions to `BeaconRewards`. Otherwise, it may be harder to call `receiveRewards` and `reportTerminations` for `uint256` identifiers.

Reply:
Sure, done.

Reply:
As a side not, I don't think we need to add those auxiliary functions in `ECDSABackportRewards / ECDSARewards` since in `Rewards.sol` they take `bytes32[]`. When passing addresses in `ECDSA*` contracts they already in `bytes`.

Reply:
Correct. `uint256[]` matters only for beacon rewards.